### ISSUE TITLE
Less Chattering Log? 

### ISSUE THREAD

**Issue by ghmia** [jeffaco/duplicacy-util#37]
I don't really need which chunks are uploaded and bcs it's on the same spinning hdd I think an option to keep less logs would be great Thx!

**Comment by jeffaco** [comment]
The `duplicacy-util` log files echo everything from `duplicacy` itself, plus additional information as needed. This is needed for diagnostic purposes if something goes wrong, and it's part of the fundamental design that I'm not inclined to change.

In general, I rely on the `duplicacy-util` e-mail messages, as well as screen output when run interactively (they are the same). This filters `duplicacy` output to what I feel is reasonable when all is well. This is generally just a few lines of output. I ignore the log files unless something goes wrong (which I rely on E-Mail notification for), and after a short time the log files are deleted.

Ultimately, if you feel that `duplicacy` itself is too wordy in it's logs (and, for what it's worth, I'm inclined to agree, although it never mattered much to me), you should log an issue against `duplicacy` itself. I'm not inclined to filer `duplicacy` logs in my detailed logs for fear of filtering out something critical if a problem occurred.

Hope you understand where I'm coming from here ...

**Comment by ghmia** [comment]
Understood Thx!
